packages/ilios-common/app/styles/ilios-common/components/fade-text.scss

```diff
@@ -0,0 +1,48 @@
+@use "../colors" as c;
+
+.display-text-wrapper {
+  &.is-faded {
+    max-height: 200px;
+    overflow: hidden;
+    position: relative;
+  }
+}
+
+.fade-text-control {
+  bottom: 140px;
+  height: 130px;
+  padding-top: 140px;
+  position: relative;
+}
+
+span {
+  &.draggable-object-content {
+    .fade-text-control {
+      background-image: linear-gradient(
+        to bottom,
+        transparent,
+        c.$culturedGrey
+      );
+    }
+  }
+}
+
+.course-objective-list {
```

Comment:
I'm not comfortable with this component reaching so far into the styling of other component to get its job done. There may not be a better way, but I'd like us to discuss a bit before taking this step.

Reply:
We should talk about this at our next meeting. Perhaps there's a better way to handle it, but it was the only way I could see at the time.